wp-includes/rest-api/endpoints/class-wp-rest-customize-changesets-controller.php

```diff
@@ -1140,4 +1224,20 @@ public function sanitize_datetime( $date ) {
 			) );
 		}
 	}
+
+	/**
+	 * Sanitize UUID.
+	 *
+	 * @param string $uuid UUID.
+	 * @return string|WP_Error Sanitized string / WP_Error if wrong format.
+	 */
+	public function sanitize_uuid( $uuid ) {
+		if ( ! preg_match( '/^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/', $uuid ) ) {
```

Comment:
This regex is now being copied here and `\WP_REST_Customize_Changesets_Controller::register_routes()`. Perhaps it should be a constant that the two places re-use.